docs/src/getting-started/wallets.md

```diff
@@ -9,24 +9,22 @@ Wallets are used for many important things, for instance:
 
 The SDK gives you many different ways to create wallets. Let's explore these different ways.
 
-## Creating a wallet from a Private Key
+## Creating a wallet
 
-You can use `fuel_crypto` to generate a random `SecretKey` and use it to create a wallet from it:
+A new wallet can be created from `Option<Provider>`.
 
 ```Rust
-use fuel_crypto::{SecretKey};
```

Comment:
I am not convinced by this new workflow. Plus, I don't think it was the point of #264.